frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/GitPushDialog.java

```diff
@@ -71,7 +85,13 @@ protected void doOKAction() {
   @Override
   @UIEffect
   protected Action[] createActions() {
-    return new Action[]{pushAction, getCancelAction(), getHelpAction()};
+    Action cancelAction = getCancelAction();
+    if (traverseInfoComponent != null) {
+      cancelAction.putValue(Action.NAME, "_Quit Traverse");
+      return new Action[]{skipAction, pushAction, pushActionAndQuit, cancelAction, getHelpAction()};
```

Comment:
Hmmm

<img width="516" alt="image" src="https://user-images.githubusercontent.com/3383210/225751181-a4f45ab2-7852-4c8b-9ed9-042611551ea8.png">

Is this button ordering consistent with the other traverse dialogs? 🤔

Reply:
`TraverseStepConfirmationDialog`:

```java
@Untainted String[] options = {yesText, yesAndQuitText, noText, quitText};
```

Reply:
Also, let's make the naming consistent (in particular, `Quit Traverse` is better than `Quit the interactive Traverse`)

Reply:
<img width="832" alt="image" src="https://user-images.githubusercontent.com/19799111/227902638-42f38208-709d-48dd-af3b-feffab18b60f.png">